bin/autosign-validator

```diff
@@ -2,9 +2,6 @@
 require 'autosign'
 require 'logging'
 
-### Read all the arugments https://tickets.puppetlabs.com/browse/SERVER-1116
-argf_read = ARGF.read
```

Comment:
It would be cleanest to just change this to replace with with the `raw_csr = $stdin.read` lines from below. No need to futz with `ARGF`.

None of the other changes would be necessary, I think.

Reply:
@danielparks I have just added another commit to switch it back to $stdin.read. 

Do you also have access to review/merge this: https://github.com/danieldreier/puppet-autosign/pull/29